src/components/Tip.tsx

```diff
@@ -55,7 +52,7 @@ export default function Tip({ total, setTip }) {
             onChangeText={(tipPercentageInput) => {
               const tipPercentageInputInt = parseInt(tipPercentageInput)
               if (isTipValid(tipPercentageInputInt)) {
-                changePercentage(total, tipPercentageInputInt, true);
+                changeTipValues(total, tipPercentageInputInt)
```

Comment:
does this need to also set the custom tip to true?

Reply:
No, this <TextInput/> component would not be visible if custom tip is not already set to `True` (see L48). Therefore, at this point, custom tip is already set to true.